### PR TITLE
first stab at a federation page

### PR DESCRIPTION
**Pull request by jospoortvliet**
Let's start working on #544 ...

Here is a first draft!

snapshot_tj9541

The image is a gif:
federation

yes, it is quite slow, that was intentional. Don't want to create a flashing page, do we?

@jancborchardt because you might want to say "try without the background gradient", here's a shot without that...

snapshot_xm9541

Let me know what you think, @karlitschek @jancborchardt @schiesbn

### CODE REVIEW

**Comment by karlitschek** [comment]
i like it :+1:

**Comment by jancborchardt** [comment]
No, the background is great! :dancer: 

Few things:
* the address in the gif is different on almost every screenshot. Just use `guy@fawkes.org/owncloud` for every one of them
* do not use "I" or "my" in body copy. Simply cut the »How do I share blabla« header. The top header suffices
* say »personal settings« instead of »user settings«. That’s how it’s called in the interface
* cut the horizontal line. The whitespace is enough

**Comment by jospoortvliet** [comment]
Done. If that is all - we can merge this whenever you guys want.

File: page-federation.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="row col-md-12">
+	<h2 class="conf-location">Share accross ownClouds!</h2>
```
Comment (LukasReschke) [inline page-federation.php]: Word says `across` and Oxford Dictionary tends to agree.

File: page-federation.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+</div>
+<div class="row conf-info">
+	<div class="col-lg-4 col-md-5 text-center">
+		<img src="<?php echo get_template_directory_uri(); ?>/assets/img/federation/federation.gif" alt="in action" />
+	</div>
+	<div class="col-lg-8 col-md-7">
+		<p>Sharing files accross ownCloud servers is as easy as sending an email from a GMail address to a Yahoo account! Instead of an email address, you use a Federation ID. And just like email, it looks like <strong>username@server</strong>, although there often is a string like 'ownCloud' attached at the end, like <strong>guy@fawkes.org/myowncloud</strong></p>
```
Comment (LukasReschke) [inline page-federation.php]: Word says `across` and Oxford Dictionary tends to agree.

Comment (LukasReschke) [inline page-federation.php]: Proper name would be `Gmail` as well. Lower-case m.

**Comment by jospoortvliet** [comment]
@LukasReschke all done, thanks for the reading.

**Comment by jancborchardt** [comment]
@jospoortvliet looks good except one thing: What are the images `federated_sharing_1.png` and `federated_sharing_2.png`, and why do they still have this strange name?

**Comment by jospoortvliet** [comment]
@jancborchardt they can be removed, they were part of the gif sources. Will remove them and merge.

**Pull request merged**